src/content/articles/alab.md

```diff
@@ -88,30 +88,47 @@ downloadSection: {
   # The download links and button setup for the download table.
   downloads: [{
     buttonText: "DOWNLOAD",
-    downloadUrl: "https://dpel-assets.aswf.io/usd-alab/alab-v2.1.0.zip",
+    downloadUrl: "https://dpel-assets.aswf.io/usd-alab/alab-v2.2.0.zip",
     size: "9.3 GB",
-    description: "USD Scene - v.2.1.0",
+    description: "",
+    descriptionBold: "Asset Structure | ",
+    extraDescription: "Main asset structure for the ALab showcasing how all assets relate to each other through USD composition arcs. This is purely the USD structure linking all files, and does not include any geometry, shaders or lights.",
```

Comment:
It might be nice to link to the GitHub repo, as an alternate way to get this component.  "Either click here to download the zip, or click here to clone the GitHub repository"

Reply:
We previously tried building from a branch other than `main` to a different URL to test deployments ([see the `publish` branch](https://github.com/AcademySoftwareFoundation/dpel-website/blob/publish/src/utils/constants.js)), but it didn't work out so we reverted to local test deployments.

Regarding links in the download table, hopefully @skomatireddy or Dima can chime in.  If not, it should at least be possible to add the GitHub repo context and link in the main asset description a bit further up the page, where we link to animallogic . com and Slack